pages/docs/tools/streamsets.html

```diff
@@ -0,0 +1,102 @@
+title: Building Data Stream pipelines with Crate and StreamSets Data Collector
+author: Ruslan Kovalov
+
+[Crate][1] can be integrated with [StreamSets][2] using its [JDBC driver][11]
+(> 1.11.0).
+
+The Crate JDBC driver can be used as an origin and destination in a StreamSets
+Data Collector pipeline.
+
+[Install the StreamSets Data Collector][3] on your localhost or other desired
+location.
+
+In order to build your first pipeline for a data stream the standalone version
+of the Crate JDBC driver with included dependencies has to be obtained and
+installed in the StreamSets.
+
+You can download the latest standalone version directly from the
+[Bintray Repository][4].
+
+After the JDBC driver is downloaded to the arbitrary destination, you can
+proceed with the installation of the driver. We recommend following the
+StreamSets instruction for installing [additional drivers][5].
+
+The next two sections provide a brief introduction into building simple data
+stream pipelines using Crate with StreamSets Data Collector. As an example,
+we demonstrate projects showing how to read data from and write to Crate
+database. To keep the manual short, we would refer to basic StreamSets tutorial
+explaining how to [configure the directory origin][6] for reading
+[CSV sample data][7], which would be further used as an input data stream in
+a pipeline with the Crate JDBC destination.
+
+## Crate JDBC Destination
+
+The Crate JDBC Producer destination uses a JDBC connection to write data to the
+database. In this section, we show how to build a data stream for inserting CSV
+records from local FS into Crate database with preliminary preprocessing
+of certain records fields.
+
+1. Create a new [pipeline][8]. 
+![Create a new pipeline]({% media '/media/1605/1-streamsets-dest-new.png' %})
+
+2. Follow the above mentioned basic
+[tutorial][6] which explains how to create the directory origin and read CSV
+files from local FS. Use data sample provided in the tutorial.
+
+3. For the sake of simplicity, we use only 4 fields from the CSV file. The
+directory origin does not perform any preliminary preprocessing of records.
+All fields of records are represented as strings. Such as selected fields
+we choose for inserting into DB are float values, we add additional
```

Comment:
However, some of the selected fields should be processed and converted into float values. So we add an additional preprocessing pipeline ...